src/main/java/de/unijena/cheminf/mortar/model/fragmentation/algorithm/CDKExhaustiveFragmenter.java

```diff
@@ -0,0 +1,283 @@
+                        GuiUtil.guiExceptionAlert(Message.get("Fragmenter.IllegalSettingValue.Title"),
+                                Message.get("Fragmenter.IllegalSettingValue.Header"),
+                                anException.toString(),
+                                anException);
+                        //re-throws the exception to properly reset the binding
+                        throw anException;
+                    }
+                    super.set(newValue);
+                }
+                else {
+                    IllegalArgumentException anException = new IllegalArgumentException("The minimum fragment size can not be zero");
+                    CDKExhaustiveFragmenter.LOGGER.log(Level.WARNING, anException.toString(), anException);
+                    GuiUtil.guiExceptionAlert(Message.get("Fragmenter.IllegalSettingValue.Title"),
+                            Message.get("Fragmenter.IllegalSettingValue.Header"),
+                            anException.toString(),
+                            anException);
+                    //re-throws the exception to properly reset the binding
+                    throw anException;
+                }
+            }
+        };
+        this.settingNameTooltipTextMap.put(this.minimumFragmentSize.getName(),
+                Message.get("CDKExhaustiveFragmenter.minFragmentSize.tooltip"));
+        this.settingNameDisplayNameMap.put(this.minimumFragmentSize.getName(),
+                Message.get("CDKExhaustiveFragmenter.minFragmentSize.displayName"));
+        this.settings = new ArrayList<>(tmpNumberOfSettings);
+        this.settings.add(this.minimumFragmentSize);
+    }
+    //</editor-fold>
+    //
+    //<editor-fold desc="Public properties get">
+    /**
+     * Returns the minimum fragment size currently set.
+     *
+     * @return the currently set minimum fragment size.
+     */
+    public int getMinimumFragmentSize() {
+        return this.minimumFragmentSize.get();
+    }
+    //</editor-fold>
+    //
+    //<editor-fold desc="IMoleculeFragmenter methods">
+    @Override
+    public List<Property<?>> settingsProperties() {
+        return this.settings;
+    }
+
+    @Override
+    public Map<String, String> getSettingNameToTooltipTextMap() {
+        return this.settingNameTooltipTextMap;
+    }
+
+    @Override
+    public Map<String, String> getSettingNameToDisplayNameMap() {
+        return this.settingNameDisplayNameMap;
+    }
+
+    @Override
+    public String getFragmentationAlgorithmName() {
+        return CDKExhaustiveFragmenter.ALGORITHM_NAME;
+    }
+
+    @Override
+    public String getFragmentationAlgorithmDisplayName() {
+        return Message.get("CDKExhaustiveFragmenter.displayName");
+    }
+
+    @Override
+    public FragmentSaturationOption getFragmentSaturationSetting() {
+        //TODO: there is currently no possibility to implement saturation settings for the exhaustive fragmenter.
```

Comment:
Can you describe this in more detail here like you did below?
And the way to do it here would be to throw an UnsupportedOperationException in these three methods. But please test whether this impacts running MORTAR. Since you do not add the property to the list of settings, we should be fine.